tests/data/assets/sel_eth_usdc_1h_9_28.csv

```diff
@@ -1,5 +1,6 @@
 Open,High,Low,Volume,Close
-7.48,7.525751740320689,7.411781931951506,959.2957,7.521012749315605
```

Comment:
Why did this change? isn't this historical data?

Reply:
This is synthetic data (see the `sel_` prefix), and it changed because we have a different random generation for the class so it has a different output